cmds.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "github.com/Azure/azure-docker-extension/pkg/vmextension"
+
+type cmdFunc func(vmextension.HandlerEnvironment) error
+
+var (
+	cmds = map[string]struct {
+		f             cmdFunc // associated function
```

Comment:
The compiler checks this code.

```go
extension.install()
```

It does not check this code.  You have to do it.

```go
m["install"].f()
```